### PR TITLE
Call out serialization support in README

### PR DESCRIPTION
**Pull request by jeremy-w**
Also adds headings to the Deserialization section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +135,53 @@ extension Person: JSONDecodable {
 
 `Person` just has a few properties. It conforms to `JSONDecodable` via an extension. In the extension, we implement a `throws`ing initializer that takes an instance of `JSON` as its sole parameter. In the implementation, we `try` three functions: 1) `string(_:)`, 2) `int(_:)`, and 3) `bool(_:)`. Each of these works as you have seen before. The methods take in a path, which is used to find a value of a specific type within the `JSON` instance passed to the initializer. Since these paths could be bad, or the requested type may not match what is actually inside of the `JSON`, these methods may potentially throw an error.
 
+
+### Serialization
+Freddy's serialization support centers around the `JSON.serialize()` function.
```
Comment (mdmathias) [inline README.md]: Can we call this a method to make it clear that `serialize()` is something you call on an instance of `JSON`?

**Comment by mdmathias** [comment]
:+1: once the Travis checks are happy.

**Comment by mdmathias** [comment]
Ooh, and may be worth referencing #129

**Comment by randomstep** [comment]
:+1: from me as well. I had to dig around to figure this out originally.

**Comment by jeremy-w** [comment]
@mdmathias I have a `[fixes #129]` in the body of the second commit message, but we've tagged it from this PR discussion now, too, so this should be very discoverable from there.

**Comment by mdmathias** [comment]
Aha, so you do.  Cool beans.  :)

**Pull request merged**